app/apps/address/views.py

```diff
@@ -4,7 +4,10 @@
 
 from apps.cases.models import Case
 from apps.homeownerassociation.models import HomeownerAssociation
-from apps.address.serializers import StatusSerializer, AddressSerializer
+from apps.address.serializers import (
+    AddressSerializer,
+    MijnAmsterdamSerializer,
```

Comment:
Was only doubting if this should be called something more specific like `MijnAmsterdamAddressSerializer` (or anything in that direction)?